src/foam/nanos/auth/User.js

```diff
@@ -336,6 +336,7 @@ foam.CLASS({
       },
       javaCode: `
         if ( ! SafetyUtil.isEmpty(getOrganization()) ) return getOrganization();
+        if ( ! SafetyUtil.isEmpty(getBusinessName()) ) return getBusinessName();
```

Comment:
please update the javascript to match. 
**suggestion for line 335**
`       return this.organization || this.businessName || ( this.lastName ? this.firstName + ' ' + this.lastName : this.firstName );
`

Reply:
😳 😰 😣 😢sorry i missed it